.github/workflows/release-step-3.yml

```diff
@@ -218,6 +218,8 @@ jobs:
           command: ""
       - name: Print Release Notes
         id: print_release_notes
+        # note: release notes here will be copied as-is from 'CHANGELOG.next-release.md'
+        # the 'pre-post-release' workflow executed after this will reset contents of 'CHANGELOG.next-release.md'
```

Comment:
[for reviewer] here the github release changelog needs to be in pure markdown, so we can't append our custom markdown from breaking changes.